test/NuGet.Server.Core.Tests/SemanticVersionJsonConverterTests.cs

```diff
@@ -48,7 +51,7 @@ public void SerializesSemanticVersionAsString(string version)
                 converter.WriteJson(writer, SemanticVersion.Parse(version), new JsonSerializer());
 
                 // Assert
-                Assert.Equal("\"" + version + "\"", json.ToString());
+                Assert.Equal("\"" + expected + "\"", json.ToString());
```

Comment:
wow.. I think luckily that all the cases for version and expected were the same this was never noticed.